code/game/objects/items/stacks/stack.dm

```diff
@@ -234,7 +234,9 @@
 	return
 
 /obj/item/stack/AltClick(mob/living/user)
-	if(user.incapacitated())
+	var/speciescheck = !ishuman(user) || !isdrone(user)
```

Comment:
To be fair I don't think anything outside of humans and drones are going to have to utilize the alt click splitting feature anyways.  There are other ways of splitting stacks after all. If, for some reason, a true devil decides to start up a building project.